tests/foreman/ui/test_host.py

```diff
@@ -1203,6 +1203,37 @@ def test_positive_manage_table_columns(
             assert (column in displayed_columns) is is_displayed
 
 
+@pytest.mark.tier2
+def test_all_hosts_manage_columns(target_sat, function_org, new_host_ui):
+    """Verify that the manage columns widget changes the columns appropriately
+
+    :id: 5e13267a-68d2-451a-ae00-6502dd5db7f4
+
+    :expectedresults: Through the widget you can change the columns on the All Hosts page
+
+    :CaseComponent:Hosts-Content
+
+    :Team: Phoenix-subscriptions
+    """
```

Comment:
We could add the RFE-to-test mapping.
```suggestion
    
    :Verifies: SAT-19064
    """
```